beheer.md

```diff
@@ -22,6 +22,40 @@ In dit document worden allerlei procedures en overzichten beschreven die van bel
 ## Lijst van deelnemende partijen en personen
 Bij de ontwikkeling van Open API standaarden nemen diverse partijen deel. Het is handig voor ontwikkelaars maar ook voor beheerders om daar een overzicht van te hebben. Daarom dient de project manager of scrum master bij de aanvang van een ontwikkelproject maar ook tijdens het project als de teamsamenstelling wijzigt de [lijst met Deelnemende partijen](https://github.com/VNG-Realisatie/api-beheer/blob/master/Deelnemende%20partijen.md) in te vullen. Ook de beheerder of beheerders van de standaard moeten na in beheername van een standaard of bij wijziging van de beheerder op deze lijst worden vermeldt.
 
+## Beheerproces
+Zodra een Open API standaard is ontwikkeld en goedgekeurd moet het worden overgedragen aan een beheerteam.
+Voor die betreffende standaard wordt dat beheerteam daarna verantwoordelijk voor:
+1. beantwoorden van vragen;
+2. het oplossen van problemen.
+
+Doorontwikkeling van de standaard hoort daar dus niet bij.
```

Comment:
Goed om expliciet te benoemen anders moet je straks 100 API's + implementaties uit je hoofd kennen :)

Reply:
Ik neem aan dat je bedoelt dat het goed is dat ik benoemd heb dat de doorontwikkeling van de standaard niet bij het beheer hoort. Je beodlet niet dat ik nog iets moet opnemen.